src/modules/tokens/views/Transfers/Transfers.tsx

```diff
@@ -182,7 +182,7 @@ export const Transfers: () => JSX.Element = () => {
               address={tokenTransfer.to}
             />
           ) : (
-            '---'
+            '--'
```

Comment:
should these be put in the translation file...just to remove hard coded strings?

Reply:
Yea, probably a good idea